src/components/structures/ThreadView.tsx

```diff
@@ -130,7 +130,7 @@ export default class ThreadView extends React.Component<IProps, IState> {
     };
 
     private setupThread = (mxEv: MatrixEvent) => {
-        let thread = mxEv.getThread();
```

Comment:
I would advise filing an issue to track fixing `getThread`, as otherwise it's likely to be forgotten.

Reply:
https://github.com/vector-im/element-web/issues/19612